src/cli/backup/backup.go

```diff
@@ -63,6 +53,48 @@ func AddCommands(cmd *cobra.Command) {
 	}
 }
 
+// ---------------------------------------------------------------------------
+// adding flags to cobra commands
+// ---------------------------------------------------------------------------
+
+// ==============================================
+// Folder Object flags
+// These options are flags for indicating
+// that a time-based filter should be used for
+// within returning objects for details.
+// Used by: OneDrive, SharePoint
+// ================================================
+var (
+	fileCreatedAfter   string
+	fileCreatedBefore  string
+	fileModifiedAfter  string
+	fileModifiedBefore string
+)
+
+// list output filter flags
+var (
+	failedItemsFN    = "failed-items"
```

Comment:
these definitions don't seem to match with the CHANGELOG update above

Reply:
whoops, thanks!